change/@ni-nimble-components-30881da7-4a97-48ad-81c0-db79e1c01c98.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I almost suggested deleting the `labelHeight` token since it's no longer used within Nimble but it's still used in [one place in Skyline](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/src/sl-details-panel/sl-details-item-custom-property/sl-details-item-custom-property.component.scss). Probably worth a quick look at that code to see if your newfound label best practices would lead us to do it differently too.

Reply:
I think it is worth keeping the token around in case we need to size something _other_ than a label to have the same height as a label. I've changed the documentation for the token in Storybook to say that you should not explicitly set the height of a label, but set the font instead.

I confirmed that setting the label height is unnecessary for the error label to be the proper height (16px). I will remove that use of the token in a follow-up PR -- at the same time I remove the workaround that Milan mentions in his comment.